core/capabilities/fakes/http_action.go

```diff
@@ -20,7 +20,7 @@ var _ httpserver.ClientCapability = (*DirectHTTPAction)(nil)
 var _ services.Service = (*DirectHTTPAction)(nil)
 var _ commonCap.ExecutableCapability = (*DirectHTTPAction)(nil)
 
-const HTTPActionID = "http-action@1.0.0"
+const HTTPActionID = "http-actions@0.1.0"
```

Comment:
is this the new id used in the sdk?

Reply:
It seems like. That's what a freshly-built workflow asks for...